src/injector/stage1/README.md

```diff
@@ -15,5 +15,17 @@ sudo apt install build-essential -y
 # build w/ musl
 rustup target add x86_64-unknown-linux-musl
 cargo build --target x86_64-unknown-linux-musl --release
-strip target/x86_64-unknown-linux-musl/release/zarf-injector
+```
+
+## Checking Binary Size
```

Comment:
Can you add a quick line that explains why we're checking the size and what to potentially do with that information?